CMakeLists.txt

```diff
@@ -82,6 +74,15 @@ add_executable("${PROJECT_NAME}"
   src/mstch.cpp
 )
 
+target_compile_options("${PROJECT_NAME}"
+  PUBLIC "--std=c++11"
```

Comment:
I've seen `-std=c++11` (note the single dash) is used to enable C++11 rather than double dashes.